src/expr.cpp

```diff
@@ -806,7 +813,7 @@ ExprParser::Token ExprParser::PopOperand(std::string *error) {
 }
 
 ExprParser::Token ExprParser::PopOperator(std::string *error) {
-    Token t = Token::From();
+    Token t;
```

Comment:
Don't think so, I think I was just explicitly calling constructor for readability. Should I revert this change?

Reply:
i FIXED IT NOW In case of error a "proper" `TokenType::ERROR` with `*expr = NULL` is returned. It probably does not matter but this was the old behaviour.